x/supply/spec/02_client.md

```diff
@@ -0,0 +1,131 @@
+<!--
+order: 2
+-->
+
+# Client 
+
+## CLI
+
+A user can query the `supply` module using the CLI. 
+
+### Query 
+
+The `query` commands allow users to query the `supply` state. 
+
+```
+desmos query supply --help
+```
+
+#### total
+The `total` command allows users to query the total supply of a token given a denomination. 
```

Comment:
```suggestion
The `total` command allows users to query the total supply of a token given a denomination and an optional divider exponent. Based on the divider exponent, the total supply is displayed differently.
```

Reply:
Instead of putting this here, since the divider exponent thing is related to all queries, I've put a note right inside the `Query` section